base/special/trig.jl

```diff
@@ -310,11 +310,28 @@ cosc(x::Complex{<:AbstractFloat}) = x==0 ? zero(x) : oftype(x,(cospi(x)-sinpi(x)
 cosc(x::Complex) = cosc(float(x))
 cosc(x::Real) = x==0 || isinf(x) ? zero(x) : (cospi(x)-sinpi(x)/(pi*x))/x
 
-for (finv, f) in ((:sec, :cos), (:csc, :sin), (:cot, :tan),
-                  (:sech, :cosh), (:csch, :sinh), (:coth, :tanh),
-                  (:secd, :cosd), (:cscd, :sind), (:cotd, :tand))
+for (finv, f, finvh, fh, finvd, fd, fn) in ((:sec, :cos, :sech, :cosh, :secd, :cosd, "secant"),
+                                          (:csc, :sin, :csch, :sinh, :cscd, :sind, "cosecant"),
+                                        (:cot, :tan, :coth, :tahnh, :cotd, :tand, "cotangent"))
```

Comment:
Thanks for the feedback! I aligned the second and third tuples as such because I'm just already over the 92 character line length with the current formatting. In that case, do you think I should leave as is, or still realign?